index.js

```diff
@@ -71,6 +71,8 @@ async function run() {
         log.replace(/I\[\d{2}-\d{2}\|\d{2}:\d{2}:\d{2}\.\d{3}\] /g, '')
       );
     },
+    unsafeRpc: true,
```

Comment:
Can we put this behing a cli flags? :)

Reply:
yes. However, I think I will switch it false and have a separate API for readonly flipper. Reason: we need to expose tendermint RPC to the world (otherwise, other nodes won't be able to connect to network), and we don't want to expose unsafe RPC

Reply:
confused p2p and RPC endpoints. We only need to expose the former. This will allow to hide RPC from the public #197